src/Microsoft.Diagnostics.NETCore.Client/Microsoft.Diagnostics.NETCore.Client.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>Library</OutputType>
-    <TargetFrameworks Condition="'$(DotNetBuildSourceOnly)' != 'true'">netstandard2.0;net6.0</TargetFrameworks>
+    <TargetFrameworks Condition="'$(DotNetBuildSourceOnly)' != 'true'">netstandard2.0;$(NetCoreAppMinTargetFramework)</TargetFrameworks>
     <TargetFramework Condition="'$(DotNetBuildSourceOnly)' == 'true'">netstandard2.0</TargetFramework>
```

Comment:
Is it intentional? My understanding is that source-build cares less about netstandard compat than non-sourcebuild (which only matters if we want .net framework compat; name of this project suggests otherwise; .NETCore.Client).

Reply:
NETCore.Client needs to support the lowest .NET version (net6.0) - sourcebuild has trouble with this since it doesn't have the 6.0 refpack available always.